model/src/main/java/org/projectnessie/model/Content.java

```diff
@@ -59,6 +60,10 @@ public String getId() {
     return UUID.randomUUID().toString();
   }
 
+  @Value.Redacted
+  @JsonIgnore
+  public abstract Type getType();
```

Comment:
Could you add a comment that we're exposing this for filtering purposes, not for serialization but that the type value is the same as what is serialized in JSON?